test/utils/expectOutput.js

```diff
@@ -20,37 +19,29 @@ module.exports = function (outputDir) {
     }
 
     const webpackConfig = args.config
+    if (webpackConfig.plugins && webpackConfig.plugins[0].options.path) {
+      outputDir = path.join(__dirname, `../../${webpackConfig.plugins[0].options.path}`)
+    }
```

Comment:
I'm not very happy about this `if` block. Neither the condition nor the way I update the `outputDir`.
It works, but I feel like it's a bit hacky and might break if more elaborate tests are added at some point.

I would love some suggestion on how to make this work more like it actually does inside webpack and the Plugin.

Reply:
@henrikhermansen, I think this is a good start. I can certainly think about this more later and make it less brittle if necessary.